stub/src/main/java/io/grpc/kotlin/Helpers.kt

```diff
@@ -75,3 +77,14 @@ internal fun <T> Flow<T>.singleOrStatusFlow(expected: String, descriptor: Any):
  */
 internal suspend fun <T> Flow<T>.singleOrStatus(expected: String, descriptor: Any): T =
   singleOrStatusFlow(expected, descriptor).single()
+
+/**
+ * Returns gRPC Metadata.
+ */
+suspend fun grpcMetadata(): Metadata {
+  val metadataElement = coroutineContext[MetadataElement]
+    ?: throw Status.INTERNAL
+      .withDescription("gRPC Metadata not found in coroutineContext. Ensure that MetadataCoroutineContextInterceptor is used in gRPC server.")
```

Comment:
There is StatusException.

Reply:
I mean StatusException like:

<img width="1562" height="596" alt="Image" src="https://github.com/user-attachments/assets/cc147301-d0e1-47da-8c95-51671f78baf0" />